web/src/features/fireWeather/components/WxDataDisplays.tsx

```diff
@@ -89,41 +87,36 @@ export const WxDataDisplays = React.memo(function _(props: WxDataDisplaysProps)
             !observations && !noonForecasts && !gdpsModels && !hrdpsModels && !rdpsModels
 
           return (
-            <Paper key={code} className={classes.paper} elevation={3}>
-              <Typography className={classes.station} variant="subtitle1" component="div">
-                Weather station: {`${station.name} (${station.code})`}
+            <div key={code} className={classes.display}>
```

Comment:
What's your reason for the move away from paper? (I know I avoid it when I can get away with just normal html)